public/app/features/dashboard/components/SaveDashboard/useDashboardSave.tsx

```diff
@@ -46,10 +46,17 @@ export const useDashboardSave = (dashboard: DashboardModel) => {
       // important that these happen before location redirect below
       appEvents.publish(new DashboardSavedEvent());
       notifyApp.success('Dashboard saved');
-      reportInteraction(`grafana_dashboard_${dashboard.id ? 'saved' : 'created'}`, {
-        name: dashboard.title,
-        url: state.value.url,
-      });
+      if (isCopy) {
+        reportInteraction('grafana_dashboard_copied', {
+          name: dashboard.title,
```

Comment:
nit: this still gives the original dashboard's name (from which the new dashboard is created).
But really I don't mind as I don't use the prop 😛

Reply:
ah... yeah refactoring to get that new name might be a little tricky so i'm inclined to just not bother if we don't need it 🤔